apps/etterlatte-behandling/src/test/kotlin/behandling/BehandlingFactoryTest.kt

```diff
@@ -203,7 +203,7 @@ class BehandlingFactoryTest {
                 persongalleri,
                 datoNaa.toString(),
                 Vedtaksloesning.GJENNY,
-            )!!.behandling
+            )!!.also { it.action() }.behandling
```

Comment:
Helt umulig å skjønne hva som skjer her uten å sjekke definisjonen av `it` og hva `action` utffører i dette tilfellet.

Reply:
Sant, denne metoden fortente eit meir spesifikt namn.